### PR TITLE
Thuang material UI setup

### PR DESCRIPTION
**Pull request by tihuan**
1. Add `@material-ui/core`
1. Add material ui storybook support for themes
1. Add README to remind developers to inject our default theme and font (Open Sans)
1. Update Button stories to include default, primary, and secondary
1. add `lint-staged`

Screen Shot 2021-03-08 at 3 55 03 PM

demo

### CODE REVIEW

File: src/core/styles/ThemeProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import React from "react";
+import defaultTheme from "./common/defaultTheme";
+
+const ThemeProvider = (props: ThemeProviderProps) => {
+  const { theme } = props;
+
+  return <RawThemeProvider {...props} theme={theme || defaultTheme} />;
+};
+
+export default ThemeProvider;
```
Comment (seve) [inline src/core/styles/ThemeProvider.tsx:14]: Why define our own `ThemeProvider`? Where will this be used?  If we use it here why not ask consumers to use it in our README

File: package.json [hunk 2]
```diff
@@ -34,12 +35,15 @@
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-sonarjs": "^0.6.0",
+    "husky": "^5.1.3",
+    "lint-staged": "^10.5.4",
```
Comment (seve) [inline package.json:39]: Are we still ditching lint staged for pr level linting

File: src/core/styles/ThemeProvider.tsx (hunk 1 above)
Comment (tihuan) [reply src/core/styles/ThemeProvider.tsx]: ah thank you! This file is now deleted 😆

File: package.json (hunk 2 above)
Comment (tihuan) [reply package.json:39]: oh great catch! Just set it up 🤩 !

**Comment by tihuan** [comment]
Added PR lint! Husky v5 has a different setup. So learned something new lol

**Review by seve — APPROVED** [review]

**Review by ovalenzuela19 — APPROVED** [review]
LGTM

**Comment by tihuan** [comment]
Thanks all! Just merged 🕺 🎉 ✨ !

**Pull request merged**